common/persistence/execution_manager_test.go

```diff
@@ -263,3 +268,277 @@ func TestGetReplicationTasks(t *testing.T) {
 		})
 	}
 }
+
```

Comment:
tests have failed on CI https://github.com/uber/cadence/actions/runs/8382150830/job/22955320462?pr=5809

Reply:
Good catch.
The issue was caused by not setting UTC in time.Unix. Fixed.